projects/core/src/configurator/commons/store/effects/configurator-place-order-hook.effect.ts

```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { Observable } from 'rxjs';
+import { map, take } from 'rxjs/operators';
+import { ActiveCartService } from '../../../../cart/facade/active-cart.service';
+import { PLACE_ORDER } from '../../../../checkout/store/actions/checkout.action';
+import { GenericConfigurator } from '../../../../model/generic-configurator.model';
+import { GenericConfigUtilsService } from '../../../generic/utils/config-utils.service';
+import { RemoveConfiguration } from '../actions/configurator.action';
+
+@Injectable()
+export class ConfiguratorPlaceOrderHookEffects {
+  @Effect()
+  placeOrder$: Observable<RemoveConfiguration> = this.actions$.pipe(
+    ofType(PLACE_ORDER),
+    map(() => {
+      const ownerKeys = [];
+      this.activeCartService
+        .getEntries()
+        .pipe(take(1))
+        .subscribe((entries) => {
+          entries.forEach((entry) => {
+            if (!entry.product?.configurable) {
```

Comment:
This check also needs to consider the configurator type, otherwise it covers the textfield entries as well e.g.

Reply:
Done, added testdata to unit test